### ISSUE TITLE
bug with the extra abillity

### ISSUE THREAD

**Issue by umbraqueen** [StygianTheBest/StygianCore#44]
Extra abillity generated when creating a char vanish upon log off. This feature is only available if you enable it in the custom login section.

**Comment by StygianTheBest** [comment]
Thanks for the report.

- Which ability is it that's disappearing when you log back on?
- Are you logging off normally or just closing the client?
- How long are you staying in the game world before logging off?

**Comment by umbraqueen** [comment]
its all the new abillity that are generated at the char creation to help with solo play.

sprint, remove curse, fake dead, etc.

all those abillity vanish when i log out
im logging out normaly, returning to character creation.
i have been like 10- 15 min in the game with my char

**Comment by StygianTheBest** [comment]
Ok, I've confirmed the issue on my end. At first, I thought it might be the spell id I chose for each, but after further testing it doesn't matter. If I learn the same spell in game using the .learn command the spell/ability will persist after logoff, but for some reason when the module invokes the same command the spells do not persist. I'm looking into a solve. When I find it I'll put up a patched worldserver.exe for download.
